boostsec/registry_validator/shared.py

```diff
@@ -1,10 +1,33 @@
 """Shared components between validation & uploads."""
-from dataclasses import dataclass
 from pathlib import Path
+from typing import Optional
 
+from pydantic import BaseModel, Field
 
-@dataclass
-class RegistryConfig:
+
+class RuleModel(BaseModel):
+    """Representation of a scanner rule."""
+
+    categories: list[str]
+    description: str
+    group: str
+    name: str
+    pretty_name: str
+    ref: str
+
+
+Rules = dict[str, RuleModel]
+
+
+class RulesDbModel(BaseModel):
+    """Representation of a rules db file content."""
+
+    imports: Optional[list[str]] = Field(None, alias="import")
+    rules: Optional[Rules]
+    default: Optional[Rules] = None
```

Comment:
Is there a reason for setting one to None and not the other?
I think only `default: Optional[Rules]` without `= None` is enough, pydantic will set it to None automatically.

Reply:
The only reason that I know of is that it makes the init of the class have a defalut value, so the lsp won't warn that a field is missing, but it's not necessary. 

I'll remove it for default!